Dockerfile

```diff
@@ -13,11 +14,27 @@ LABEL summary="Sumologic Kubernetes Collection Helm Operator for the Sumo Logic
 LABEL description="Sumologic Kubernetes Collection Helm Operator deploys https://github.com/SumoLogic/sumologic-kubernetes-collection"
 
 USER root
-RUN microdnf --setopt=install_weak_deps=0 install yum && yum -y update
-USER ${USER_UID}
+
+RUN microdnf update && microdnf install -y tar gzip wget && microdnf clean all
+
+RUN wget "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz" && \
+    tar -xzvf "helm-v${HELM_VERSION}-linux-amd64.tar.gz" && \
+    mv linux-amd64/helm /usr/local/bin/helm
 
 ENV HOME=/opt/helm
 COPY licenses /licenses
 COPY watches.yaml ${HOME}/watches.yaml
 COPY helm-charts/sumologic-kubernetes-collection/deploy/helm/sumologic  ${HOME}/helm-charts/sumologic
+
+# patch for Telegraf Operator Helm chart
+# can be removed when Telegraf Operator is updated in sumologic kubernetes collection to version with
+# https://github.com/influxdata/helm-charts/pull/349
+WORKDIR ${HOME}/helm-charts/sumologic
+RUN helm dependency update && cd charts && for subchart in *.tgz; do tar -xf "${subchart}" && rm -f "${subchart}"; done;
+RUN sed -i "s#{{ .Values.image.repository }}:{{ .Chart.AppVersion }}#{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}#g" charts/telegraf-operator/templates/deployment.yaml
+
+# patch for prometheus specification to use sha in image path
+COPY patches/kube-prometheus-stack/12.3.0/prometheus.yaml charts/kube-prometheus-stack/templates/prometheus/prometheus.yaml
```

Comment:
I'd rather use the `patch` command for changes visibility but it's OK like that as well